src/components/withLocalize.tsx

```diff
@@ -30,24 +30,30 @@ const withLocalizePropTypes = {
     toLocaleDigit: PropTypes.func.isRequired,
 };
 
-export default function withLocalize(WrappedComponent) {
-    const WithLocalize = forwardRef((props, ref) => (
-        <LocaleContext.Consumer>
-            {(translateUtils) => (
-                <WrappedComponent
-                    // eslint-disable-next-line react/jsx-props-no-spreading
-                    {...translateUtils}
-                    // eslint-disable-next-line react/jsx-props-no-spreading
-                    {...props}
-                    ref={ref}
-                />
-            )}
-        </LocaleContext.Consumer>
-    ));
+type WithLocalizeProps = LocaleContextProps;
```

Comment:
is that needed ?

Reply:
It is just for convenience when using `withLocalize`:
```ts
type ComponentProps = WithLocalizeProps & {
    something: string;
};

function Component(props: ComponentProps) {
    const addressLine = props.translate('common.addressLine', {lineNumber: 2});
    return (
        <Text>
            {addressLine}
            {props.something}
        </Text>
    );
}

const WrappedComponent = withLocalize(Component);

<WrappedComponent something="something" />;
```